config.rb.example

```diff
@@ -33,6 +33,8 @@ module Config
   HEADER = 'WiFi @ Home'
   HEADER_IMAGE = File.expand_path('./data/fox.png', __dir__)
 
+  ROOT_CA_URL = 'https://somewhere.fox/rootca.crt'
```

Comment:
please add a comment saying something like `set this to nil to hide the CA message`